### ISSUE TITLE
[Feature Request] Auto Download with previous chapter too

### ISSUE THREAD

**Issue by EyangKodok** [inorichi/tachiyomi#3611]
Can we change the "auto download when there is new update" thing to "auto download when there is new update THE new chapter AND the previous chapter, because some translation took like a month or so, reader might already forgot the plot and what not, which we will end up need to read a chapter or two before new one to get the "feel" again so we might need to download the previous chapter manually which make the auto download thing obsolete imo, surely you got the idea.

Make the auto download thing to have an option to download a chapter or two before new update of each updated manga.

TL;DR Just make auto download thing to download new chapter and previous chapter.

**Comment by CarlosEsco** [comment]
No plans to add this.  Adding the extra logic to find the previous chapter is not worth it for a feature that would rarely be used.  Instead why not turn on delete after read and bump the number up to 1 or 2.  This way it keeps the last 1-2 chapters in your library
